### PR TITLE
feat(oauth): send feature flag to frontend

### PR DESCRIPTION
**Pull request by sentaur-athena**
There's no org [where this feature flag is being checked](https://github.com/getsentry/getsentry/blob/master/static/getsentry/gsAdmin/views/instanceLevelOAuth/components/newInstanceLevelOAuthClient.tsx#L24) so enabling it through the config. This basically makes it available in the Config object on the frontend

### CODE REVIEW

File: src/sentry/web/client_config.py [hunk 1]
```diff
@@ -224,6 +224,11 @@ def enabled_features(self) -> Iterable[str]:
             yield "relocation:enabled"
         if features.has("system:multi-region"):
             yield "system:multi-region"
+        # TODO @athena: remove this feature flag after development is done
+        # this is a temporary hack to be able to used flagpole in a case where there's no organization
+        # availble on the frontend
+        if self.last_org and features.has("organizations:scoped-partner-oauth", self.last_org):
```
Comment (markstory) [inline src/sentry/web/client_config.py:230]: Are you planning on using organization conditions for this feature check? The `last_org` attribute could reflect an arbitrary organization depending on where the session last visited. If you want to have user level conditions for this flag you should pass the `actor` parameter. Right now you won't be able to use user attributes.

Comment (sentaur-athena) [reply src/sentry/web/client_config.py:230]: I see. So I can pass actor and last_org but from flagpol yml ignore the org completely?

Comment (sentaur-athena) [reply src/sentry/web/client_config.py:230]: oh I see an example above. Like organization:create, just with actor.

File: src/sentry/web/client_config.py [hunk 2]
```diff
@@ -224,6 +224,11 @@ def enabled_features(self) -> Iterable[str]:
             yield "relocation:enabled"
         if features.has("system:multi-region"):
             yield "system:multi-region"
+        # TODO @athena: remove this feature flag after development is done
+        # this is a temporary hack to be able to used flagpole in a case where there's no organization
+        # availble on the frontend
+        if self.last_org and features.has("organizations:scoped-partner-oauth", actor=self.user):
```
Comment (markstory) [inline src/sentry/web/client_config.py:230]: You'll need an organization as well to have flagpole actually do a feature flag check.

**Review by markstory — APPROVED** [review]

**Pull request merged**